### DISCUSSION TITLE
Memory error: allocate E1V0

### DISCUSSION THREAD

**Discussion by rhig2000** [wenbinyugroup/.github#28] [VABS Related Discussions] [answered]
Echo returned the following error:
>Memory error: allocate E1V0

How can I allocate more memory? Any suggestions?

**Comment by unitasium** [comment]
Can you provide more details of this error? Where does it come from?

**Reply by wenbinyu-PU** [reply, marked as answer]
the best way is to send us the input file to debug.

**Reply by rhig2000** [reply]
Here is an excerpt including first and last lines

>  Format Control Parameter
>  ========================================================
>  
>  Old Input Format is using
>  
>  Control Parameters
>  ========================================================
>  
>  Timoshenko  =            1
>  Damping    =            0
>  Thermal Analysis=            0
>  Curved      =            1
>  Oblique     =            0
>  Trapeze     =            1
>  Vlasov      =            1
>  
>  Initial Curvatures/Twist
>  ========================================================
>  
>  k1=    180.000000000000     
>  k2=   0.000000000000000E+000
>  k3=   0.000000000000000E+000
>  
>  Mesh control parameters
>  ========================================================
>  
>  Nodes         =        39379
>  Elements      =        12912
>  Material Types=           34
>  
>  
>  
>  Error Message
>  ========================================================
>  
>  Memory error: allocate E1V0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     
>
